test/deploy/windows/kafka/install/roles/start/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- debug:
+    msg: Starting Zookeper and Kafka Server
+
+- name: check if batch script for zookeeper exists
+  ansible.windows.win_stat:
+    path: c:\kafka\bin\windows\zookeeper-server-start.bat
+  register: zookeeper_batch_path
+
+- name: check if batch script for kafka exists
+  ansible.windows.win_stat:
+    path: c:\kafka\bin\windows\kafka-server-start.bat
+  register: kafka_batch_path
+
+- block:
+    - name: Starting Zookeper
+      win_shell: c:\kafka\bin\windows\zookeeper-server-start.bat c:\kafka\config\zookeeper.properties
+      async: 5
+      poll: 0
+    - name: Starting Kafka Server
+      win_shell: c:\kafka\bin\windows\kafka-server-start.bat c:\kafka\config\server.properties
+      async: 5
+      poll: 0
+  ignore_errors: yes
+  when: zookeeper_batch_path.stat.exists and kafka_batch_path.stat.exists
+
+- block:
+    - name: end play when path not found
+      debug:
+        msg: "The zookeeper or kafka batch script doesn't exist in c:\\kafka\\bin\\windows, unable to start kafka"
+  when: not zookeeper_batch_path.stat.exists or not kafka_batch_path.stat.exist
+  ignore_errors: yes
```

Comment:
@NRhzhao I dont think we should ignore error if we see we can't start? I would also change the ansible debug with a fail command instead as I believe we want to error the play if we couldn't start?

Reply:
@Julien4218 Since this task only fires when the directory doesn't exist, so it shows an error when running ansible.  I cannot remove it, but not sure that's the right way of displaying.